static/local/blackwomenssuffrage/key-figures.md

```diff
@@ -1,83 +1,199 @@
nt her to attend Philadelphia's segregated schools. Instead, at 16 years old, she was sent to the more progressive Salem, Massachusetts, where she boarded with the Remond family, who were also prominent Black abolitionists. She began keeping a diary while studying at Higginson Grammar School for Girls, where she was the only Black student. These diaries became a lifelong habit and they were posthumously published in 1988, offering insight into Black life in the final years of slavery and the time immediately after abolition. During her time in Salem, Forten became a member of the [Salem Female Anti-Slavery Society](/search?q=Salem+Female+Anti-Slavery+Society) and was active in sewing items to sell at the group's annual anti-slavery fundraising fairs, as well as a frequent attendee of anti-slavery lectures. In 1856, Forten became the first-ever Black graduate of what would become Salem State University. Wishing to support herself without the help of a husband, she became a teacher at an all white-school in Salem and began writing poetry, some of which was published in William Lloyd son's [The Liberator](/search?q=The+Liberator). Following in her family's activist footsteps, Forten moved to the Sea Islands of South Carolina in 1862 as part of an effort to educate enslaved people on plantations abandoned by their owners when Union forces gained control of the area at the beginning of the Civil War. Although she left the islands after the death of her father and repeated bouts of pneumonia in 1864, she wrote about her time there in the two-part essay "Life in the Sea Islands," which was published in Atlantic Monthly. The essay depicted a Black population eager to learn, support themselves and assume the new responsibilities of citizenship. Back in Massachusetts, she served as secretary of the [Freedmen's Union Commission's](/search?q=Freedmens+Union+Commission) Boston branch. Forten later returned to teaching in South Carolina and at an all-black school in Washington, D.C. and clerked at the U.S. Department of the Treasury. When she was 41, Forten married the Reverend Francis Grimk&eacute;, who was 13 years her junior. Grimk&eacute; was the son of a white plantation owner named Henry Grimk&eacute; and his black enslaved worker Nancy Weston. He and his brother Archibald, who would become the executive director of the [National Association for the Advancement of Colored People](/search?q=National+Association+for+the+Advancement+of+Colored+People) (NAACP), became free at the end of the Civil War. Grimk&eacute;'s education at Lincoln University was subsidized by his aunts, the white abolitionist and suffragist sisters Sarah and Angelina Grimk&eacute;. After her marriage, Forten Grimk&eacute; continued her activism, joining the [National Association of Colored Women](/search?q=National+Association+of+Colored+Women) (NACW) and advocating for women's suffrage. She also continued to write poetry and essays calling for an end to racial injustice and violence.
```

Comment:
You can add a link to "Charlotte Forten Grimke" `/search?q=Charlotte+Grimke` (no accent over the "e" within the search term)